lib/gamera/modules/visitable.rb

```diff
@@ -28,7 +28,7 @@ def visit(fail_on_redirect: true)
     # @return [Boolean] true if the url loaded in the browser matches the url_matcher pattern
     # @raise [NoUrlMatcherForPage] if there's no url_matcher for this page
     def displayed?(wait_time_seconds = CAPYBARA_DEFAULT_MAX_WAIT_TIME)
-      fail Gamera::NoUrlMatcherForPage if url_matcher.nil?
+      raise Gamera::NoUrlMatcherForPage if url_matcher.nil?
```

Comment:
Gamera::Visitable#displayed? performs a nil-check

[Read more about it here.](https://github.com/troessner/reek/blob/master/docs/Nil-Check.md)